DEVELOPMENT.md

```diff
@@ -4,54 +4,116 @@ The Things Stack components are primarily built in Go, we use React for web fron
 
 ## Table of contents
 
-- [Development Environment](#development-environment)
-- [Cloning the Repository](#cloning-the-repository)
-- [Getting Started](#getting-started)
-- [Running a development build of The Things Stack](#running-a-development-build-of-the-things-stack)
-  - [Pre-requisites](#pre-requisites)
-  - [Steps](#steps)
-- [Using the CLI with the Development Environment](#using-the-cli-with-the-development-environment)
-- [Managing the Development Databases](#managing-the-development-databases)
-  - [PostgreSQL](#PostgreSQL)
-  - [Redis](#redis)
-- [Building the Frontend](#building-the-frontend)
-- [Starting The Things Stack](#starting-the-things-stack)
-- [Project Structure](#project-structure)
-  - [API](#api)
-  - [Documentation](#documentation)
-  - [Web UI](#web-ui)
-- [Code Style](#code-style)
-  - [Code Formatting](#code-formatting)
-  - [Line Length](#line-length)
-  - [Formatting and Linting](#formatting-and-linting)
-  - [Documentation Site](#documentation-site)
-- [Naming Guidelines](#naming-guidelines)
-  - [API Method Naming](#api-method-naming)
-  - [Variable Naming](#variable-naming)
-  - [Event Naming](#event-naming)
-  - [Error Naming](#error-naming)
-  - [Log Field Keys, Event Names, Error Names, Error Attributes and Task Identifiers](#log-field-keys-event-names-error-names-error-attributes-and-task-identifiers)
-  - [Comments](#comments)
-- [JavaScript Code Style](#javascript-code-style)
-  - [Code Formatting](#code-formatting)
-  - [Code Comments](#code-comments)
-  - [Import Statement Order](#import-statement-order)
-  - [React Component Syntax (Functional, Class Components and Hooks)](#react-component-syntax-functional-class-components-and-hooks)
-  - [React Component Types](#react-component-types)
-  - [Frontend Related Pull Requests](#frontend-related-pull-requests)
-- [Translations](#translations)
-  - [Backend Translations](#backend-translations)
-  - [Frontend Translations](#frontend-translations)
-- [Events](#events)
-- [Testing](#testing)
-  - [Unit Tests](#unit-tests)
-  - [End-to-end Tests](#end-to-end-tests)
-- [Building and Running](#building-and-running)
-- [Releasing](#releasing)
-  - [Release From Master](#release-from-master)
-  - [Release Backports](#release-backports)
-- [Troubleshooting](#troubleshooting)
-  - [Console](#console)
+- [The Things Stack for LoRaWAN Development](#the-things-stack-for-lorawan-development)
```

Comment:
I don't think we need this index anymore as it is now automatically generated by GitHub.